syntax = "proto3";

service Serializer {
  rpc Add(OperationRequest) returns (ResultResponse) {}
  rpc TesteSerial(SerialData) returns (SerialData) {}
}

message ResultResponse {
  double result = 1;
  string error = 2;
}

message OperationRequest {
  float a = 1;
  float b = 2;
}

message SerialData {
  string data = 1;
  bytes data2 = 2;
}
